Build failed in Jenkins: Ibin_uT64__jsk_arc2017_common__ubuntu_trusty_amd64__binary

```
make[4]: Entering directory `/tmp/binarydeb/ros-indigo-jsk-arc2017-common-3.0.0/obj-x86_64-linux-gnu'
../scripts/install_data.py
Traceback (most recent call last):
  File "../scripts/install_data.py", line 3, in <module>
    from jsk_data import download_data
ImportError: No module named jsk_data
make[4]: *** [CMakeFiles/install_data] Error 1
make[4]: Leaving directory `/tmp/binarydeb/ros-indigo-jsk-arc2017-common-3.0.0/obj-x86_64-linux-gnu'
make[3]: *** [CMakeFiles/install_data.dir/all] Error 2
make[3]: Leaving directory `/tmp/binarydeb/ros-indigo-jsk-arc2017-common-3.0.0/obj-x86_64-linux-gnu'
make[2]: *** [all] Error 2
make[2]: Leaving directory `/tmp/binarydeb/ros-indigo-jsk-arc2017-common-3.0.0/obj-x86_64-linux-gnu'
dh_auto_build: make -j1 returned exit code 2
make[1]: *** [override_dh_auto_build] Error 2
make[1]: Leaving directory `/tmp/binarydeb/ros-indigo-jsk-arc2017-common-3.0.0'
make: *** [build] Error 2
dpkg-buildpackage: error: debian/rules build gave error exit status 2
E: Building failed
Traceback (most recent call last):
  File "/tmp/ros_buildfarm/ros_buildfarm/binarydeb_job.py", line 133, in build_binarydeb
    subprocess.check_call(cmd, cwd=source_dir)
  File "/usr/lib/python3.4/subprocess.py", line 561, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['apt-src', 'build', 'ros-indigo-jsk-arc2017-common']' returned non-zero exit status 1
# END SUBSECTION
```

--- a/jsk_arc2017_common/package.xml
+++ b/jsk_arc2017_common/package.xml
@@ -18,6 +18,7 @@
   <build_depend>roseus</build_depend>
 
   <run_depend>jsk_apc2016_common</run_depend>
+  <run_depend>jsk_data</run_depend>
   <run_depend>jsk_pcl_ros</run_depend>
   <run_depend>jsk_pcl_ros_utils</run_depend>
   <run_depend>jsk_perception</run_depend>

